src/tools/perf/lib/uct_tests.cc

```diff
@@ -783,21 +783,50 @@ class uct_perf_test_runner {
    TEST_CASE_ALL_OSD(_perf, _case, UCT_PERF_DATA_LAYOUT_BCOPY) \
    TEST_CASE_ALL_OSD(_perf, _case, UCT_PERF_DATA_LAYOUT_ZCOPY)
 
+#define DISPATCH(_name, _cmd, _type) \
```

Comment:
`_name` -> `_case` (consistency w code above)

Reply:
Consider renaming `DISPATCH` and `DISPATCHER_ENTRY` so that the relation between them is clearer.
One implements a func, the other one returns its name.. Also, can reuse it.

Reply:
`DISPATCH` -> `DEFINE_DISPATCH_FUNC`
`DISPATCHER_ENTRY` -> `DISPATCH_FUNC_ENTRY`